File: core/CleanArchFramework.Domain/Entities/About.cs
using System.ComponentModel;
using CleanArchFramework.Domain.Common;

namespace CleanArchFramework.Domain.Entities
{
    public  class About : BaseEntity<int>, IAuditableEntity
    {
        public Guid? ImageId { get; set; }
        [Localizable(true)]
        public FileSet? Image { get; set; }
        //public Guid? BackgroundImageId { get; set; }
        //public FileStorage? BackgroundImage { get; set; }
        public Guid? TitleId { get; set; }
        [Localizable(true)]
        public virtual LocalizationSet? Title { get; set; }
        public Guid? SubtitleId { get; set; }
        [Localizable(true)]
        public virtual LocalizationSet? Subtitle { get; set; }
        public Guid? InfoTextId { get; set; }
        [Localizable(true)]
        public virtual LocalizationSet? InfoText { get; set; }
        public string? CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public string? ModifiedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
    }
}

File: ui/app/[lng]/_services/about.ts
import { ApiResponse } from '../_interfaces/interfaces';

const aboutApiUrl = `${process.env.API_URL}/About`;
// const token = authService.getToken();

export interface IAbout {
	imageId: string;
	dataImage: string;
	backgroundImageId: string;
	dataBackgroundImage: string;
	title: string;
	subtitle: string;
	infoText: string;
}

export interface AboutResponse extends IAbout {
	id: number;
	createdBy: string;
	createdDate: string;
	modifiedBy: string;
	modifiedOn: string;
}

interface IAboutService {
	getAbout: (lang:string) => Promise<AboutResponse>;
	updateAbout: (
		id: number,
		dataImage: File | null,
		dataBackgroundImage: File | null,
		title: string,
		subtitle: string,
		infoText: string
	) => Promise<IAbout>;
}

export const aboutService: IAboutService = {
	async getAbout(lang:string) {
		try {
			const response = await fetch(aboutApiUrl, {
				method: 'GET',
				headers: {
					'Accept-Language': lang,
				},
			});
			const result: ApiResponse = await response.json();
			return Promise.resolve(result.data as AboutResponse);
		} catch (error) {
			return Promise.reject()

			
		}
	},
	async updateAbout(
		id: number,
		image: File | null,
		backgroundImage: File | null,
		title: string,
		subtitle: string,
		infoText: string
	) {
		try {
			let data = new FormData();
			data.append('id', `${id}`);
			if (image) {
				data.append('image', image);
			}
			if (backgroundImage) {
				data.append('backgroundImage', backgroundImage);
			}
			data.append('title', title);
			data.append('subtitle', subtitle);
			data.append('infoText', infoText);

			const response = await fetch(aboutApiUrl, {
				headers: {
					// Authorization: `Bearer ${token}`,
				},
				method: 'PUT',
				body: data,
			});
			const result: IAbout = await response.json();
			return Promise.resolve(result);
		} catch (error) {
			return Promise.reject(error);
		}
	},
};
